package.json

```diff
@@ -130,7 +132,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn build"
```

Comment:
@charlieduong94 What do you think about this, I figured this can help catch things that would fail on ci before they get pushed up.

Reply:
:+1: Seems good to me